gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
+networkTimeout=10000
+validateDistributionUrl=true
```

Comment:
I noticed that this is an "incubating" feature. I suppose that means it's subject to change. Any issues you can foresee with that?

Reply:
Huh. Those were generated when I did `gradle wrapper` on the latest Gradle release. I can just remove the two extra lines because I don't see the added value if they're incubating.